index.js

```diff
@@ -72,9 +72,11 @@ var browserslist = function (selections, opts) {
         selections = selections.split(/,\s*/);
     }
 
-    if ( opts.stats || process.env.BROWSERSLIST_STATS ) {
+    var statsFile;
+
+    var getStats = function () {
```

Comment:
@ai ESLint thinks it's a complexity in this case, trying come up with a solution